var/spack/repos/builtin/packages/heffte/package.py

```diff
@@ -165,6 +165,9 @@ def test(self):
                 "-Drocfft_DIR=" + join_path(self.spec["rocfft"].prefix, "lib", "cmake", "rocfft")
             )
 
+        # Provide the path to the MPI dependency so that the test can find an MPI launcher
+        options.append("-DCMAKE_PREFIX_PATH=" + self.spec["mpi"].prefix)
```

Comment:
I have this working in #35750 with the following change:

```suggestion
        options.append("-DMPI_HOME=" + self.spec["mpi"].prefix)
```

Reply:
Well,  technically working when I use these changes with #34236 .

Reply:
@tldahlgren is right, `MPI_HOME` is the correct thing to do here.

@G-Ragghianti can you update the PR, please?